crowbar_framework/app/views/nodes/_show.html.haml

```diff
@@ -18,7 +18,7 @@
       = render :partial => "show_barclamps"
       = render :partial => "show_roles"
 
-  - if @node.bmc_set? or not @node.admin?
+  - if (@node.bmc_set? or not @node.admin?) && @node.state != "crowbar-upgrade"
```

Comment:
This cries for another node object method. Pretty complicated condition for a view.